src/selenium.ts

```diff
@@ -76,7 +76,7 @@ async function seleniumStart(
 
   const config: selenium.StartOpts = SELENIUM_OVERRIDES || {};
   config.seleniumArgs = ['-port', port.toString()].concat(opts.args);
-  config.seleniumJavaArgs = [];
+  config.javaArgs = [];
```

Comment:
So, may be call it consistently seleniumJavaArgs?

Reply:
But will it work? I see in selenium it's javaArgs and seleniumArgs. So maybe we should be consitent to make sure. Otherwise this won't work.